blind_walking/net/feature_encoder.py

```diff
@@ -6,6 +6,28 @@
 from stable_baselines3.common.torch_layers import BaseFeaturesExtractor
 from stable_baselines3.common.torch_layers import create_mlp
 
+def _create_depthwise_separable_conv2d_block(in_channels, out_channels):
```

Comment:
move to a utilities/utils folder if you want to keep it

Reply:
done